### ISSUE TITLE
Load LoRA weight into original Hunyuan weight

### ISSUE THREAD

**Issue by Aristo23333** [a-r-r-o-w/finetrainers#166]
Hi author:
Thanks for your finetuning method. I have successfully fine-tune Hunyuan model. I notice that the hunyuan official huggingface provides a 25G weight of transformer https://huggingface.co/tencent/HunyuanVideo/tree/main. And your huggingface style diffusions are divided  into 6 parts. Can the finetrainers loRA weight be integrated into the original Hunyuan weight?

**Comment by sayakpaul** [comment]
Yes, that should be ready to go. If you face difficulties, then please open an issue in https://github.com/huggingface/diffusers
